packages/devtools_app/lib/src/screens/performance/panes/rebuild_stats/rebuild_stats_model.dart

```diff
@@ -68,10 +68,6 @@ const _eventsKey = 'events';
 class LocationMap {
   LocationMap();
 
-  factory LocationMap.fromJson(Map<String, dynamic> json) {
```

Comment:
@jacob314 should this be removed? or is this just not used yet since this feature is under construction?

Reply:
should be removed